recipes/winpcap/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans import ConanFile, CMake
+import os
+import shutil
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        for file_name in ["1_packet.pcap", "wpcap.dll", "Packet.dll"]:
```

Comment:
Adding binary files is pretty bad. I think the old test_package might be better.

A test_package only needs to check if the Conan package was created correctly, not test internal logic